src/utils/dataset-helper/get-files-differences.js

```diff
@@ -114,11 +106,10 @@ export default (objectOfRef, objectToCheck) => {
     else if (objectToCheck.mode !== objectOfRef.mode) resultSet.mode = true
   }
 
-  // Check if the fileName doesn't exist or have change
-  if (!isEmpty(objectToCheck.fileName)) {
-    if (isEmpty(objectOfRef.fileName)) resultSet.fileName = true
-    else if (objectToCheck.fileName !== objectOfRef.fileName)
-      resultSet.fileName = true
+  // Check if the name doesn't exist or have change
+  if (!isEmpty(objectToCheck.name)) {
+    if (isEmpty(objectOfRef.name)) resultSet.name = true
+    else if (objectToCheck.name !== objectOfRef.name) resultSet.name = true
```

Comment:
I think this type of logic should probably be moved to the dataset manager, because it seems like it's a component of managing a dataset.

Reply:
You're right #It's a part of my old manager

Reply:
Maybe should I change it for an event onChange so I will forget it for this pr